src/engine/SCons/Node/__init__.py

```diff
@@ -637,8 +637,10 @@ def get_executor(self, create=1):
         """Fetch the action executor for this node.  Create one if
         there isn't already one, and requested to do so."""
         try:
-            executor = self.executor
-        except AttributeError:
+            if self.executor is None:
```

Comment:
It looks like the exception won't be thrown based on your previous comments about slots usage. Additionally I think it should only create the Null executor if create is True?

Reply:
I agree with the create argument.
this fixes the msvc test. I had some code that this also fixes. .. let me tweak.